.gitignore

```diff
@@ -1,8 +1,10 @@
 node_modules
-settings.json
-project.geany
-timer_settings.json.bak
-MHTimer.log
-MHTimer.pid
+/settings.json
+/project.geany
+/timer_settings.json.bak
+/MHTimer.log*
+/MHTimer.pid
 .test_timers.json
 .timers.json
```

Comment:
I think all "." files should be excluded no matter the directory structure. But putting a / makes sense.